scheds/rust/scx_layered/src/main.rs

```diff
@@ -1291,20 +1408,14 @@ impl Layer {
         // Do we already have enough?
         if nr_cpus >= cpus_min
             && (layer_util == 0.0 || (nr_cpus > 0 && layer_util / nr_cpus as f64 <= util_high))
+            && ((nr_cpus - free) as f64 / cpu_pool.nr_cpus as f64 >= layer_load / total_load)
```

Comment:
This might not be correct because now the layer load is the infeasible adjusted load.